src/components/footer/FooterCenter.scss

```diff
@@ -1,10 +1,11 @@
-.footer-center {
+.layer-ui__wrapper__footer-center {
```

Comment:
reverted this naming change to align with our naming styleguide (or rather, sibling components). It was changed here https://github.com/excalidraw/excalidraw/pull/6034/files#diff-d72e8343b3e4f5407e64d2612320f2f2d1f004e365c13df5dfb861c771875ab0L8 and I don't recall why (I found no comment in the PR)

Reply:
why do we need `layer-ui__wrapper` here? css top level class name should be as file name which in this case `FooterCenter` hence `footer-center` is more appropriate here

Reply:
To align with the rest of the components. I'd change the styleguide in one go instead of having mixed naming scheme — which we already have now, but at least related components should use the same scheme.

Furthermore, `footer-center` doesn't seem right since it's a sub-component of `Footer`.

Reply:
Footer is the parent component so there should be a top level class `footer` and the sub components should start with the `footer-center`, `footer-top` etc same as comp / file name convention. 

Naming it differently with some top-level parent is more confusing so I think we should keep it as is and refactor later as needed.